### DISCUSSION TITLE
Keep background image static when resizing

### DISCUSSION THREAD

**Discussion by tjnaz** [wezterm/wezterm#2924] [Q&A] [answered]
So, I've been using Wezterm for almost 6 months now and to be honest it's the perfect terminal emulator for me.

Anyway the problem I'm having is with my wallpaper, not exactly an issue strictly speaking, however it's more of personal requirements which I can't seem to find an answer at the moment.

I want the background image to be not resized along with the window, I want it to stay static.

1. Is it possible?
2. If so, how do I go about doing it?
3. If not, may it's possible to create a feature request in the github repo?

Please let me know if I need to include any further details.

That's all.

**Comment by wez** [comment, marked as answer]
Take a look at https://wezfurlong.org/wezterm/config/lua/config/background.html which has a number of more advanced options for controlling the background image layer(s).

I think you probably want to set the width and height to your desired fixed size of your image, and then the horizontal and vertical alignment to center and middle.

**Reply by tjnaz** [reply]
Thanks, that did the trick.
